apps/condo/domains/miniapp/components/AppSelector/AppSelectPageContent.tsx

```diff
@@ -179,25 +182,30 @@ export const AppSelectPageContent: React.FC = () => {
                                         {
                                             isAnyBillingAvailable && (
                                                 <Tabs.TabPane tab={BillingCategoryMessage} key={'billing'}>
-                                                    <CardsContainer cardsPerRow={appsPerRow}>
-                                                        {
-                                                            unconnectedBillingsApps.map(app => {
-                                                                const tag = intl.formatMessage({ id: `miniapps.category.${app.category}` })
-                                                                const url = `/miniapps/${app.id}/about?type=${app.type}`
-                                                                const logo = app.logo || undefined
-                                                                return (
-                                                                    <AppSelectCard
-                                                                        key={app.name}
-                                                                        title={app.name}
-                                                                        description={app.shortDescription}
-                                                                        url={url}
-                                                                        tag={tag}
-                                                                        logoSrc={logo}
-                                                                    />
-                                                                )
-                                                            })
-                                                        }
-                                                    </CardsContainer>
+                                                    <AmplitudeAuthorizedUser>
+                                                        <TrackPageLoadEvent eventType={AmplitudeEventType.VisitBillingAppListPage}>
```

Comment:
From the name of `TrackPageLoadEvent` it states, like it should be used only inside of pages, but actually it's used inside components. Take a look at my proposal to utilize already existing `PageWrapper` component.